components/ciphertool/src/main/java/org/wso2/ciphertool/utils/Utils.java

```diff
@@ -414,6 +495,51 @@ public static Map<String, String> getSecreteFromConfiguration(String configFileP
         return context;
     }
 
+    /**
+     * Read from toml file and return keystore data.
+     *
+     * @param configFilePath    file path to deployment toml

```

Comment:
Update the comments

Reply:
Addressed by https://github.com/wso2/cipher-tool/pull/76